src/lib.rs

```diff
@@ -333,6 +333,21 @@ where
     runner.run(f);
 }
 
+/// Run the given function under a random seed for one iteration.
```

Comment:
It isn't clear what `under a random seed for one iteration` means. Also the `one iteration` part is wrong. We run the function `f` under a `RandomScheduler` initialized with the provided `seed` for `iterations` iterations.

Reply:
oops, good catch -- I forgot to update the comment